includes/batch.form.inc

```diff
@@ -47,14 +47,20 @@ function islandora_newspaper_batch_form($form, &$form_state, $object) {
     if ($derive['ocr']) {
       $form['generate_ocr'] = array(
         '#type' => 'checkbox',
-        '#title' => t('Generate OCR?'),
-        '#description' => t('Whether or not we should generate OCR for the pages of the newspaper issue.'),
+        '#title' => t('Generate OCR'),
+        '#description' => t('Generate HOCR for the pages of each newspaper issue.'),
```

Comment:
This needs some little fixing: says OCR, but text says HOCR

Reply:
Whoops! Changin'